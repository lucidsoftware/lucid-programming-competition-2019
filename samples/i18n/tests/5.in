100
iablrbvjukuryzmefmc
hitgjnypwwvsueqyxufoxaliljqec
bgrewynyepk
plswfkqvrev
oghotcqnnijcs
lfgpjyafgk
rxhvfwzwfirnmnncjiczergwryxplb
bpklpjxfjxemfokqzonit
rjfhbinkfqm
cqcycnjnkurmvyadymqmg
qnngehbqzuc
dapozymriciapptmqztwpmfv
ghfocvmxxfxhdhajcrqx
egsrycdxarxbjtuojiowcoaswgh
dubnultdbynlrvqiux
qcxmcxdlfcjik
gnegnhtpueijmfrpofdli
xqmsayuovsujsz
wnpxogcmziayv
aqzyipuvncpxbbowhosk
eevkyfeqm
memiesrsft
qsfpmw
wevthjl
ifw
oeyg
wsu
nmgwqlmfibizolebjbsfoqdnf
clslzdoeqnsffome
livxyywaucenuiwosozlrcaz
oahqwoty
iwcbbelloyheudswpoasfjnxypm
evikxaorhqhh
yiqw
oqv
awtwsuebiemhtez
zktvnwbqhcjismaziqndkj
abltttgcacwmyrloenrsrmzex
vftqyixmkakpao
kon
xrahholhsbvvyivi
awotnnhtcyyuylswmftxfykccimjau
iimhzrtqzdftzbsdnhcfo
tvstko
jxelujcorggezmyieqoafjaawfksln
nhfmkhxywrkfrsjuqly
qhbclrkbyz
hbmjfdoiosvljcsnrudvsqu
lq
nifznahvyclzmhahr
jtnm
cdaufjrvzyxzpxoxnzyams
nal
yqxdbthlbtuyopw
tje
opedjkkokjpxgdnjmuyvcnk
ci
rholactbfnfyiekl
blscopvxmyimzwmuy
dgtbuyhytanyknuvbjnoquaqi
msjklmjviruhpslmyuvdttjocr
bikauhskpfukgzroo
fpceqkhjouedfxv
okiwkqetbdcnogskmwh
rkmwlvszzgmtwzg
njxbmmklhsgbohsnlextlormqtjpy
viclk
fdchiggclcmvipllekqa
dulqeifbgzdryvflfkdq
xecdmjvzafthetxmdqonyxiq
mmvxshccw
akhpncrz
rynkkc
oxnuxigdlfmrnvgbbabiiujfpekj
nfdamjv
rsqeatlxhvbgcvtsuuwyrbivajqkmt
vbyrrwb
gjlezvcnmz
srrwosdpiwimrsnys
bydjdiqfvlbmgo
tifkzetkfygoaqrzpynoyryeouoyof
bbztbipslkfsooycqxzzchqyc
xdaqdfuysspafnhrarogzxtfn
fzbshws
abayqdcis
iiiqrzgtkmgdfaj
dgpihggwajotsdtalwpso
wecsmwnqlif
lwzibowrcywoozlnkjs
bjwamzlfotzxgpesodtgrgputgg
ktqukmfjjwcdlsylfuyvuyquldkfay
lkyde
gyagtnxmneakuvxxktigxwpypplets
jtbtpkhxiwiykhlpjyhtugkie
wcqptdy
ytitqqbjqkntnfnmgmube
jbaekrr
xhzmtwnttden
dd
dzur
