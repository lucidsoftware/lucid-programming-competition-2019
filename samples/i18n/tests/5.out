i17c
h27c
b9k
p9v
o11s
l8k
r28b
b19t
r9m
c19g
q9c
d22v
g18x
e25h
d16x
q11k
g19i
x12z
w11v
a18k
e7m
m8t
q4w
w5l
i1w
o2g
w1u
n23f
c14e
l22z
o6y
i25m
e10h
y2w
o1v
a13z
z20j
a23x
v12o
k1n
x14i
a28u
i19o
t4o
j28n
n17y
q8z
h21u
lq
n15r
j2m
c20s
n1l
y13w
t1e
o21k
ci
r14l
b15y
d23i
m24r
b15o
f13v
o17h
r13g
n27y
v3k
f18a
d18q
x22q
m7w
a6z
r4c
o26j
n5v
r28t
v5b
g8z
s15s
b12o
t28f
b23c
x23n
f5s
a7s
i13j
d19o
w9f
l17s
b25g
k28y
l3e
g28s
j23e
w5y
y19e
j5r
x10n
dd
d2r
